.github/workflows/legacy-test.yml

```diff
@@ -0,0 +1,152 @@
+name: Legacy App Testing
+on:
+  schedule:
+    - cron: "0 0,12 * * *"
```

Comment:
set it to run twice a day, is that okay?

Reply:
LGTM